test/accessibility.html

```diff
@@ -145,11 +145,21 @@
           content.dispatchEvent(e);
         });
 
-        it('should focus the editor on esc', done => {
+        it('should prevent keydown and focus the editor on esc', done => {
           sinon.stub(editor, 'focus', done);
           const e = new CustomEvent('keydown', {bubbles: true});
           e.keyCode = 27;
-          buttons[0].dispatchEvent(e);
+          const result = buttons[0].dispatchEvent(e);
+          expect(result).to.be.false;
```

Comment:
Nit: not sure, but probably a good idea to add a note that the return value is false if event is cancelable and some handler called `preventDefault()`.

Reply:
Done.